server/copy.go

```diff
@@ -59,3 +66,129 @@ func copyFile(srcFile, dstFile string, wr WriterFunc) error {
 
 	return nil
 }
+
+func copyFromFileStore(dst, src string, b filestore.FileBackend) error {
+	if ok, err := b.FileExists(src); !ok {
+		return nil
```

Comment:
Not sure if it's important, but it's possible for `FileExists` to return false with a "unable to know if file %s exists` error, but then we wouldn't return the error here -- is that ok?

Reply:
Good point, it's actually okay as we will continue to the loop where we call the copy function.